cmd/ucpd/ucp-dev.yaml

```diff
@@ -54,6 +54,9 @@ ucp:
   direct:
     endpoint: "http://localhost:9000/apis/api.ucp.dev/v1alpha3"
 
+routing:
+  defaultDownstreamEndpoint: "http://localhost:8082"
```

Comment:
We can add a comment saying that this address is dynamic-rp's address.

Reply:
Sure